culture/trust_levels.md

```diff
@@ -0,0 +1,108 @@
+systems built by the SourceCred contributors but rather the composition
+of a community:
+
+  - **Trust level 3** (TL3): Collaborative. It is taken as [common
+    knowledge][common-knowledge] that everyone in the community is
+    acting in good faith. People feel comfortable reflecting on their
+    own incentives: “am I subconsciously skewing my ‘votes’ to benefit
+    myself?”; “how have my motivations for votes, etc. changed over
+    time?”. When asking the same questions of others, we trust that the
+    responses are thoughtfully and truthfully given. Discussions of
+    accidental or subconscious gaming are treated as precious data to be
+    shared and analyzed, never as evidence of any kind of wrongdoing.
+
+  - **Trust level 2** (TL2): Wary. While the community’s core backbone
+    of contributors are still dedicated to the original ideals, perhaps
+    some new members are more interested in expanding their share of the
+    pie rather than the pie as a whole. Primarily distinguished from TL3
+    because good faith is no longer a uniform assumption. Expect skewed
+    voting records, underrecognition of others’ help, quarrels over
+    attribution or allocation.
+
+  - **Trust level 1** (TL1): Alert. This community controls a scarce
+    resource. That resource is considered intrinsically valuable by some
+    people outside the community, who are willing to devote their time
+    to acquiring that resource without much thought for scruples.
+    Primarily distinguished from TL2 because gaming may now be the main
+    goal rather than an opportunistic “side hustle” on legitimate
+    contributions—noting, of course, that legitimate components may yet
+    be a key component of effective gaming strategies! Expect voting
+    cliques, [sybils], chains of obfuscated self-dealing, attempts to
+    corrupt structures of leadership and governance.
+
+[common-knowledge]: https://en.wikipedia.org/wiki/Common_knowledge_(logic)
+[sybils]: https://en.wikipedia.org/wiki/Sybil_attack
+
+(As a point of syntax, note that the numbers decrease as the community
+is understood to be less trustworthy. That is, the number indicates the
+degree of trust, not the passage of time. This is for simplicity that
+“high trust level should mean more trust”, as well as consistency with
+systems like DEFCON or Discourse’s own trust levels.)
+
+# Usage
+
+It’s important to recognize that trust level varies both with time and
+with community. As of late 2019, the SourceCred community is in TL3, but
+(e.g.) some cryptocurrency communities are surely in TL1.
+
+A given iteration of the SourceCred system will be appropriate for some
+subset of trust levels. As of late 2019, SourceCred is probably only
+appropriate for communities in TL3. As we begin to service communities
+at lower trust levels—perhaps including our own at a later time—we’ll
+need to anneal down, investing into systems that can handle the
+corresponding heat.
```

Comment:
Any reason to not update to 2020, since it's being written here now?

Reply:
Per @decentralion's [nit for approval](https://github.com/sourcecred/docs/pull/24#pullrequestreview-366767991), 
suggest rewording this para to:

>It’s important to recognize that trust level varies both with time and with community. As of early 2020, SourceCred has only been actively tested with communities in TL3, is probably not appropriate for TL1, and should be used in TL2 with caution and close attention.